spec/dummy/app/api/api.rb

```diff
@@ -26,5 +26,10 @@ class API < Grape::API
     request.params.as_json
   end
 
+  desc 'Get Authorization header.', authorizations: { oauth2: [] }
```

Comment:
It would be nice if this actually did OAuth2 :) I would rename the endpoint either way to `oauth2`.